src/main/kotlin/com/sourcegraph/cody/autocomplete/CodyAutocompleteManager.kt

```diff
@@ -155,6 +157,11 @@ class CodyAutocompleteManager {
     if (isTriggeredImplicitly && !autoCompleteDocumentContext.isCompletionTriggerValid()) {
       return
     }
+    if (isTriggeredExplicitly &&
+        CodyAuthenticationManager.instance.getActiveAccount(project) == null) {
+      HintManager.getInstance().showErrorHint(editor, "Cody: No account signed-in")
```

Comment:
Hmm, I'm not sure what is the best phrase. How about this?
```suggestion
      HintManager.getInstance().showErrorHint(editor, "Cody: sign in to use autocomplete")
```

cc/ @danielmarquespt any ideas? This error is only displayed on the explicit alt-backslash shortcut